### PR TITLE
[New] add qoopido.demand w/ single package.json

### PR DESCRIPTION
**Pull request by dlueth**
Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/dlueth/qoopido.demand
 * Official website (optional, not the repository): https://demand.qoopido.com/
 * NPM package url (optional): https://www.npmjs.com/package/qoopido.demand
 * License and its reference:
 * GitHub / Bitbucket popularity (required): MIT OR GPL-3.0+
   - Count of stars: 15
   - Count of watchers: 3
   - Count of forks: 1
 * NPM download stats (optional):
   - Downloads in the last day: 9
   - Downloads in the last week: 802
   - Downloads in the last month: 1213

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/qoopido.demand/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "qoopido.demand",
+  "filename": "demand.js",
+  "version": "4.0.7",
```
Comment (PeterDaveHello) [inline ajax/libs/qoopido.demand/package.json]: Please remove "version" field since you didn't really commit its files.

Comment (dlueth) [inline ajax/libs/qoopido.demand/package.json]: done, wondering why the ci fails though

File: ajax/libs/qoopido.demand/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dlueth/qoopido.demand"
+  },
+  "license": "(MIT OR GPL-3.0+)"
```
Comment (pvnr0082t) [inline ajax/libs/qoopido.demand/package.json]: Let's use licenses array as following:
```json
  "licenses": [
    "MIT",
    "GPL-3.0"
  ],
```

Comment (dlueth) [inline ajax/libs/qoopido.demand/package.json]: done

**Comment by dlueth** [comment]
added author to package.json

File: ajax/libs/qoopido.demand/package.json [hunk 3]
```diff
@@ -34,5 +38,8 @@
     "type": "git",
     "url": "https://github.com/dlueth/qoopido.demand"
   },
-  "license": "(MIT OR GPL-3.0+)"
+  "licenses": [
+    "MIT",
+    "GPL-3.0"
```
Comment (PeterDaveHello) [inline ajax/libs/qoopido.demand/package.json]: I think it should be `"deprecated_GPL-3.0+"` in SPDX format.

https://github.com/cdnjs/cdnjs/blob/12d79a4bf555f80bd5b8c1b694ccf127e69aea0c/tools/license-list.json#L119

Comment (dlueth) [inline ajax/libs/qoopido.demand/package.json]: corrected the license - you are right, should be deprecated_GPL-3.0+

**Pull request merged**